lib/public/components/detector/DetectorSelectionDropdownModel.js

```diff
@@ -13,7 +13,7 @@
 
 import { SelectionDropdownModel } from '../common/selection/dropdown/SelectionDropdownModel.js';
 import { RemoteData } from '/js/src/index.js';
-import { detectorsProvider } from '../../services/detectors/detectorsProvider.js';
+import { createDetectorsProviderInstance } from '../../services/detectors/detectorsProvider.js';
```

Comment:
Why do you prefer this instead of a singleton?

Side note: I think this change do not belongs to this PR in any case.

Reply:
https://github.com/AliceO2Group/Bookkeeping/pull/1288#discussion_r1424232887